formats/specialist_document/frontend/examples/aaib-reports.json

```diff
@@ -10,6 +10,7 @@
 			],
 			"report_type" : "correspondence-investigation",
 			"date_of_occurrence" : "2015-08-08",
+			"bulk_published" : false,
```

Comment:
It needs added to each of the metadata definitions [here](https://github.com/alphagov/govuk-content-schemas/blob/bulkpublishedfix/dist/formats/specialist_document/frontend/schema.json#L136-L163).